Manual/GAP_code/vecspaces_example/complete_example.g

```diff
@@ -20,14 +23,18 @@ DeclareOperation( "SQVecObj",
 DeclareOperation( "SQVecMor",
                   [ IsSQVecObj, IsObject, IsSQVecObj ] );
 
-BindGlobal( "SQVec", CreateCapCategory( "Skeletal category of rational vector spaces" ) );
+BindGlobal( "SQVec", CreateCapCategoryWithDataTypes(
+                             "Skeletal category of rational vector spaces",
+                             IsSQVecCat,
+                             IsSQVecObj,
+                             IsSQVecMor,
+                             IsCapCategoryTwoCell,
+                             fail,
+                             fail,
+                             fail ) );
```

Comment:
I suggest to simply use `CreateCapCategory` to avoid having to talk about the `fail`s.

Reply:
Done.